mono/utils/mono-context.h

```diff
@@ -33,6 +33,7 @@ typedef struct __darwin_xmm_reg MonoContextSimdReg;
 typedef struct _libc_xmmreg MonoContextSimdReg;
 #elif defined(HOST_WIN32)
 #define MONO_HAVE_SIMD_REG
+#define MONO_HAVE_SIMD_REG_AVX
 #include <emmintrin.h>
```

Comment:
Unrelated

Reply:
I had copied mono-context.h from dotnet/runtimelab as a lazy way of picking up my s390x changes. I will revert thost two pieces of code to their pre-MONO_AVE_SIMD_REG_AVX state with my s390x changes then added.